app/components/telegram-auth.tsx

```diff
@@ -177,6 +178,7 @@ function TwoFAForm({
 }
 
 export default function TelegramAuth() {
+  const { logLoginStarted, logLoginSuccess } = useAnalytics()
```

Comment:
does it make sense to make this log specific to the Telegram login flow? Something like `logLoginTgStarted`?
Since we support both Telegram and Storacha logins, and users with an existing plan might skip the Humanode step